esp-config/src/generate/value.rs

```diff
@@ -6,11 +6,12 @@ use super::Error;
 
 /// Supported configuration value types.
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(untagged)]
 pub enum Value {
     /// Booleans.
     Bool(bool),
     /// Integers.
-    Integer(i128),
+    Integer(i64),
```

Comment:
i128 was intentional, why do we need to change it?

Reply:
Yaml has problems with i128/u128 (i.e. doesn't support it) - I know we want a biggest possible range but i64 sounds quite reasonable for a 32 bit target?